source/features/copy-file.tsx

```diff
@@ -1,26 +1,32 @@
 import React from 'dom-chef';
 import select from 'select-dom';
 import copyToClipboard from 'copy-text-to-clipboard';
+import delegate from 'delegate-it';
 import features from '../libs/features';
 
 function handleClick({currentTarget: button}: React.MouseEvent<HTMLButtonElement>): void {
 	const file = button.closest('.Box');
-
 	const content = select.all('.blob-code-inner', file!)
 		.map(blob => blob.innerText) // Must be `.innerText`
 		.map(line => line === '\n' ? '' : line)
 		.join('\n');
-
 	copyToClipboard(content);
 }
 
-function init(): void {
-	// This selector skips binaries + markdowns with code
-	for (const code of select.all('.blob-wrapper > .highlight:not(.rgh-copy-file)')) {
-		code.classList.add('rgh-copy-file');
-		code
-			.closest('.Box')! // Closest common container
-			.querySelector('[data-hotkey="b"]')! // Easily-found `Blame` button
+async function init(): Promise<void> {
+	// This won't show button on markdown and binary files until the code is loaded
+	if (features.isMarkDown() || select.all('.blob-wrapper > .highlight').length === 0) {
```

Comment:
Since we're listening to `rgh:view-markdown-source`, we need to match its condition:

```suggestion
	if (select.exists('.blob.instapaper_body')) {
```